core/src/test/scala/caliban/RenderingSpec.scala

```diff
@@ -11,8 +11,11 @@ object RenderingSpec extends DefaultRunnableSpec {
   override def spec: ZSpec[TestEnvironment, Any] =
     suite("rendering")(
       test("it should render directives") {
-        assert(graphQL(resolver).render.trim)(
-          equalTo("""schema {
+        assert(graphQL(resolver, directives = List(Directives.Test)).render.trim)(
+          equalTo(""""Test directive"
```

Comment:
I've been playing around with graphql-java, and one thing they do for their schema render is they render the built-in directives (`@skip`, `@include`, `@deprecated`, `@specifiedBy`) by default in every schema, even if they are not specified. Did we have any interest in doing this? I think [they may be optional according to the spec](https://spec.graphql.org/October2021/#sel-FAHnBPLCAACCcooU)? But might be a nice thing to do?

Reply:
I don't have a strong opinion about it. It's a little polluting in simple cases but could be useful in some. @paulpdaniels @frekw any opinions?

Reply:
It might be annoying for the tests if it's always included. Maybe an optional parameter?

Reply:
> It might be annoying for the tests if it's always included. Maybe an optional parameter to `render`?

I have another PR https://github.com/ghostdogpr/caliban/pull/1219 that I'm gonna work on updating the `render()` API. I'll look at adding this functionality when I tackle that I think if that sounds good?

Reply:
Sure!